src/presentation/cells/support.ts

```diff
@@ -99,33 +95,32 @@ export function focusOnReference(modelNode: ModelNode, referenceName: string): v
     return;
   }
 
-  const inputs = $(firstNodeFound).find('input');
+  const inputs = firstNodeFound.querySelectorAll('input');
   let refNode;
-  if (
-    $(firstNodeFound).data('node_represented') === modelNode.idString() &&
-    $(firstNodeFound).data('reference_represented') === referenceName
-  ) {
-    refNode = $(firstNodeFound);
+  if (firstNodeFound.dataset.node_represented === modelNode.idString() &&
+      firstNodeFound.dataset.reference_represented === referenceName) {
+    refNode = [firstNodeFound];
   } else {
-    refNode = inputs.filter((i, el) => {
+     refNode = _.filter(inputs, (el, i) => {
```

Comment:
what is the `_` identifier?

Reply:
It's the lodash library of functional operators (was underscore.js). We were already including it so I used it.